recipes/noteslib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "noteslib" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [unix or py2k]
+
+requirements:
+  host:
+    - pip ==18.1
```

Comment:
why the pip pin?

Reply:
@CJ-Wright I was getting a backend failure `raise BackendUnavailable pip._vendor.pep517.wrappers.BackendUnavailable` in all commits before `41f6680` ...this thread https://github.com/pypa/pip/issues/6164 showed a work around by reverting back to 18.1. All else showed skipping Windows, but this package is made exclusively for Windows and was released (without further update) when pip's most recent release was 18.1